scripts/sync-workflow.js

```diff
@@ -114,54 +93,10 @@ function initialSync(sessionToken) {
   });
 }
 
-/**
- * Test step: SyncRecords request to get the data from the server
- * @param {object} previousResolution - contains session token and dataset hash
- * @returns {promise} A promise that resolves with an object
- * containing session token and dataset hash
- */
-function syncRecords(previousResolution) {
-  const sessionToken = previousResolution.sessionToken;
-  const serverHash = previousResolution.serverHash;
-  const fhRequest = configureRequest(clientIdentifier, sessionToken);
-  const reqBody = requestBodyUtils.getSyncRecordsRequestBody({
-    dataset_id: 'workorders',
-    query_params: {
-      "filter": {
-        "key": "assignee",
-        "value": "rkX1fdSH"
-      }
-    },
-    dataset_hash: serverHash,
-    meta_data: {
-      clientIdentifier: clientIdentifier
-    },
-    pending: []
-  });
-
-  return new Promise(resolve => {
-    lr.actStart('Sync Records');
-    return fhRequest.post({
-      url: `${argv.app}/mbaas/sync/workorders`,
-      body: reqBody,
-      json: true
-    }).then(resBody => {
-      lr.actEnd('Sync Records');
-
-      const resolution = {
-        sessionToken: sessionToken,
-        serverHash: resBody.hash
-      };
-      return resolve(resolution);
-    });
-  });
-}
-
-
 // Execution starts here
 login(lr, configureRequest(clientIdentifier), argv.app, argv.username, argv.password)
```

Comment:
Now that we have the setup script creating enough users for the run, this should change to using those users. So it would be something like `loaduser${argv.runNum}`, if we had a `runNum` parameter that received the value from load-runner.

Not something that necessarily needs to be done in this PR, just highlighting it while I see it! :)